### PR TITLE
Add missing KERN_INSUFFICIENT_BUFFER_SIZE mach error code

### PR DESCRIPTION
**Pull request by danra**
#### Resolved bug number: ([SR-](https://bugs.swift.org/browse/SR-))

---

Before merging this pull request to apple/swift repository:
- [ ] Test pull request on Swift continuous integration.

<details>
  <summary>

Triggering Swift CI</summary>

The swift-ci is triggered by writing a comment on this PR addressed to the GitHub user @swift-ci. Different tests will run depending on the specific comment that you use. The currently available comments are:

**Smoke Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please smoke test |
| OS X platform | @swift-ci Please smoke test OS X platform |
| Linux platform | @swift-ci Please smoke test Linux platform |

 **Validation Testing**

| Platform | Comment |
| --- | --- |
| All supported platforms | @swift-ci Please test |
| OS X platform | @swift-ci Please test OS X platform |
| Linux platform | @swift-ci Please test Linux platform |

Note: Only members of the Apple organization can trigger swift-ci.
</details>

The added code is present in kern_return.h of OS X 10.11 SDK, but not here.

### CODE REVIEW

File: stdlib/public/Platform/MachError.swift [hunk 1]
```diff
@@ -189,5 +189,8 @@
 
   /// The requested property cannot be changed at this time.
   case KERN_POLICY_STATIC             = 51
+  
+  /// The provided buffer is of insufficient size for the requested data.
+  case KERN_INSUFFICIENT_BUFFER_SIZE	52
```
Comment (compnerd) [inline stdlib/public/Platform/MachError.swift]: Missing `=`?

**Comment by danra** [comment]
@compnerd Oops, fixed.

**Comment by lattner** [comment]
@swift-ci Please test

**Comment by danra** [comment]
Looks like an unrelated error

**Comment by danra** [comment]
@lattner Can you please re-run the test? Thanks!

**Pull request closed without merging**